### DISCUSSION TITLE
纯小白，Recent Post卡片的头图开关如何控制？

### DISCUSSION THREAD

**Discussion by Hulenkius** [ppoffice/hexo-theme-icarus#1081] [Q&A] [answered]
希望达成如图效果
![image](https://user-images.githubusercontent.com/32562298/171986475-cf18ba7c-f3b5-4a1b-b488-4f4e2d104c2c.png)

**Comment by realzza** [comment, marked as answer]
在你的博客`.md`文件的header里去掉`thumbnail`
